cpp/src/aztec/honk/sumcheck/polynomials/univariate.hpp

```diff
@@ -38,8 +43,39 @@ template <class Fr, size_t _length> class Univariate {
         }
     }
 
+    // TODO(luke): I'd like to be able to construct a Univariate directly from a buffer this leads to ambiguity with the
```

Comment:
This is resolved by `serialize_from_buffer` defined below?

Reply:
Yeah, removed this commented out constructor/comment